gflownet/gflownet.py

```diff
@@ -168,7 +168,13 @@ def __init__(
         self.jsd = -1.0
 
     def _tfloat(self, x):
-        return torch.tensor(x, dtype=self.float, device=self.device)
+        if isinstance(x, torch.Tensor):  # logq
+            return x.to(self.device).type(self.float)
```

Comment:
Minor change to simplify:
```suggestion
            return x.to(self.device, self.float)
```